RecoTauTag/RecoTau/plugins/RecoTauJetRegionProducer.cc

```diff
@@ -149,6 +149,11 @@ void RecoTauJetRegionProducer::produce(edm::Event& evt, const edm::EventSetup& e
 	  pfCand != pfCands.end(); ++pfCand ) {
       bool isMappedToJet = false;
       if ( pfCandAssocMapSrc_.label() != "" ) {
+	auto temp = jetToPFCandMap->find(jetRef);
+	if( temp == jetToPFCandMap->end() ) {
+	  edm::LogWarning("WeirdCandidateMap") << "Candidate map for jet " << jetRef.key() << " is empty!";
```

Comment:
how frequent is this?
We should not tolerate more than once in ~100 events or so.